presto-hive/src/main/java/com/facebook/presto/hive/HiveErrorCode.java

```diff
@@ -50,7 +50,8 @@
     HIVE_WRITER_OPEN_ERROR(25, EXTERNAL),
     HIVE_WRITER_CLOSE_ERROR(26, EXTERNAL),
     HIVE_WRITER_DATA_ERROR(27, EXTERNAL),
-    HIVE_INVALID_BUCKET_FILES(28, EXTERNAL);
+    HIVE_INVALID_BUCKET_FILES(28, EXTERNAL),
+    HIVE_EXCEEDED_PARTITION_LIMIT(29, EXTERNAL);
```

Comment:
Good catch, will update. Separately, there are some others that should also be updated, like `HIVE_COLUMN_ORDER_MISMATCH` or `HIVE_TOO_MANY_OPEN_PARTITIONS`.